src/w2grid.js

```diff
@@ -6981,11 +7020,18 @@
         },
 
         prepareData: function () {
+            var obj = this;
+
             // loops thru records and prepares date and time objects
             for (var r = 0; r < this.records.length; r++) {
                 var rec = this.records[r];
-                for (var c = 0; c < this.columns.length; c++) {
-                    var column = this.columns[c];
+                prepareRecord(rec);
```

Comment:
The (red) lines 6987 and 6988 above are deleted.
"var rec = .." and prepareRecord() are at the same level.
GIT is showing the diff in a weird way, but i think indentation is correct.